docs/faq.md

```diff
@@ -41,3 +41,10 @@ The {ref}`reference transcriptome index <processing_information:reference transc
 The resulting reference transcriptome index contains 60,319 genes.
 In addition to protein-coding genes, this list of genes includes pseudogenes and non-coding RNA.
 The gene expression data files available for download report all possible genes present in the reference transcriptome, even if not detected in a given library. 
+
+## Where can I see the code for generating QC reports? 
+
+A QC report for every processed library is included with all downloads. 
+Following alignment and quantification, the unfiltered output from Alevin-fry was imported into R as a `SingleCellExperiment` object and filtered using `DropletUtils::emptyDrops()`. 
+Next, we generated the QC report, using the unfiltered and filtered `SingleCellExperiment` objects as input. 
+You can find the [code used to generate the QC report](https://github.com/AlexsLemonade/scpcaTools/blob/fd536edebddb2589c430c460ae7c79458dae803c/inst/rmd/qc_report.rmd) in the package we developed for working with processed ScPCA data, `scpcaTools`(https://github.com/AlexsLemonade/scpcaTools/tree/v0.1.1). 
```

Comment:
I'm thinking maybe we don't want to link to specific versions here (or elsewhere in the docs). I worry more about linking to old versions and forgetting to update than people getting a version that is not quite in sync with something that they downloaded. 

I also thought that we might just link to the function, since that is how most people would use them, if they so desired. I then linked to the directory with the templates, but I don't feel strongly about that: if we want to link to the main template as you had before, that would be fine.   
 
```suggestion
You can find the [function for generating a QC report](https://github.com/AlexsLemonade/scpcaTools/blob/main/R/generate_qc_report.R) and the [QC report template documents](https://github.com/AlexsLemonade/scpcaTools/tree/main/inst/rmd) in the package we developed for working with processed ScPCA data, `scpcaTools`(https://github.com/AlexsLemonade/scpcaTools). 
```